docs/get-started/getting-started.md

```diff
@@ -28,27 +29,23 @@ Clone the deck.gl repo, if you haven't already
 git clone git@github.com:uber/deck.gl.git
 ```
 
-For most consistent results, it is recommended that you check out the latest release branch (e.g. `git checkout 6.0-release`) instead of `master` when running examples.
+For most consistent results, it is recommended that you check out the latest release branch (e.g. `6.4-release`) instead of `master` when running examples.
 
 ```bash
-git checkout 6.0-release
+git checkout 6.4-release
```

Comment:
If this is on master, 7.0-release?

Reply:
This will be bumped when we cut the release branch using script.